cmd/thanos/config.go

```diff
@@ -45,6 +46,9 @@ func (gc *grpcConfig) registerFlag(cmd extkingpin.FlagClause) *grpcConfig {
 	cmd.Flag("grpc-server-tls-client-ca",
 		"TLS CA to verify clients against. If no client CA is specified, there is no client verification on server side. (tls.NoClientCert)").
 		Default("").StringVar(&gc.tlsSrvClientCA)
+	cmd.Flag("grpc-server-tls-min-version",
+		"TLS minimum version to gRPC server, unset will default to tls 1.3, allow values: [\"1.0\", \"1.1\", \"1.2\", \"1.3\"]").
```

Comment:
Maybe: 

"TLS supported minimum version for gRPC server. If no version is specified, it'll default to 1.3. Allowed values: [[\"1.0\", \"1.1\", \"1.2\", \"1.3\"]]"

Reply:
I have updated the description. also remove the extra square bracket. from [["1.0", "1.1", "1.2", "1.3"]] to ["1.0", "1.1", "1.2", "1.3"], thought that might be typo and confusing for reading.